docs/README.md

```diff
@@ -78,7 +92,5 @@ Facets are a feature in ElasticPress which add control to filter content by one
 
 Depending on the configuration specified for `facets`, if the `match-type` property is set to `any`, it will force the results to match any selected taxonomy term. If set to `all`, it will match to results with all of the selected terms.
 
-### Auto Suggest
-The default auto suggest functionality has been modified but does not effect the default WP search form template.
-
-In addition to the default auto suggest endpoint, an additional endpoint was added (`/autosuggest/`). This endpoint accepts as json object to modify the parameters that are forwarded to ElasticPress for suggesting posts.
+### Search Form Auto Suggest
+This feature enhances search forms on the website to show a dropdown list of suggestions as users type.
```

Comment:
Does this also need a link to some relevant documentation?

Reply:
I realised that when this was implemented the code I wrote to make this secure for Enterprise Kit wasn't actually copied over. I'm somewhat tempted to remove or replace the built in ElasticPress feature altogether to be honest. We don't have any other documentation for it right now anyway and we didn't have anything more detailed previously.